src/floyd-triangle.js

```diff
@@ -0,0 +1,13 @@
+function Floyd(n) {
+    let shape = "", k = 1;
```

Comment:
don't declare variable like this.
it's should be 
```
let shape = "";
let k = 1;
```
It's just for more clarify